### DISCUSSION TITLE
Support Outside CMS

### DISCUSSION THREAD

**Discussion by vduffel99** [CMSgov/hospital-price-transparency#152] [Q&A]
Hi Price Transparency Community. I wanted to see if there is somewhere beside here where creators of the MRF collaborate. I am interested in how other are completing specific elements of the file (for example: Why chose Tall vs Wide vs JSON) and ask question CMS cannot answer (the technical questions, not compliance). I am trying to do my best but and fall short in a couple areas. Any guidance is appreciated. Thank you.

**Comment by foustsWCH** [comment]
Hi vduffel99, I saw that you closed this ticket.  Did you find another space to collaborate/connect with other file developers or did you just give up on getting a response?  It seems like a valuable request and something that I would be interested in as well.

**Comment by vduffel99** [comment]
I did not receive any responses and gave up. A lot of my questions go unanswered on the Github and so I got discouraged. I do agree that it was a valid request and I understand (and respect) that there are questions CMS cannot answer. Which is why collaborating with peers would be amazing. I would like to share what I am doing and hear what others are doing.

Victoria Duffel MAFM, CRCR, CPC, CFE
Managing Consultant / Healthcare Performance Improvement
D: 816.489.4259
***@***.***
forvismazars.us<https://www.forvismazars.us/>

Follow Forvis Mazars on LinkedIn<https://www.linkedin.com/company/forvis-mazars-us/about/> | X<https://x.com/i/flow/login?redirect_after_login=%2FForvisMazarsUS> | Facebook<https://www.facebook.com/forvismazarsusa> | Instagram<https://www.instagram.com/forvis_llp/> | ***@***.***>

**Comment by foustsWCH** [comment]
I agree. When CMS had originally announced they were using GitHub, I had thought there would be more overall activity and user interaction and that it would be a helpful resource for learning.

I do think at this point the norm is established.  Since most questions are only answered by CMS, and most CMS responses are formal compliance responses and either (1) cite language from the final rule or (2) refer back to their example files they’ve put together, there’s not much point in posting other types of questions, since the question probably won’t get much of a response…  which means there’s not much activity on Github, which means there’s not a lot of people who could respond to the questions….  and it becomes a self-defeating spiral.

I’d assume there may be an uptick in questions in the next few weeks since the new requirements go into place in January.  **Maybe some other people will see this thread and also express interest in collaborating.** 

I’m not sure that I’m a good person to answer your CSV wide vs tall vs JSON format question, but since you asked, if it is helpful, we are using CSV-wide for our hospital.   Our file is put together with a series of SQL statements.  There is one “SELECT…FROM…WHERE” query for each code set (the “from” clause references the codes i.e. chargemaster, MS-DRG, formulary, etc. and any joined tables to calculate the rates).  The column order/names are the same for each query and include all contracted payers even if rates are not applicable for that payer for that code type.  At the end, the queries are all joined together with a UNION ALL statement (to stack them “on top” of each other) to make a final file.  It's probably not the most efficient approach, but it works.

As far as the reasons why CSV wide was selected: 
(1)	CSV/excel is a more familiar format for our managed care team than JSON, so it is easier to review and validate.
(2)	The wide format doesn’t repeat the code description/modifier/etc. information a bunch of times like the tall format, which is redundant and feels clunky. 
(3)	The wide format creates “blank spaces’ where rates are not applicable.  This could be a drawback for some users since it creates a lot of white space.  However, it gives a good visual of what is present/missing from the file (i.e. should commercial payer A have an established DRG rate or is it intentionally blank?, etc.), which feels a bit harder to easily "see" in the tall format.
Not very sophisticated reasons.  I'd be curious what others are doing.

**Comment by vduffel99** [comment]
I agree, I thought it would be a forum where we all would offer information but since we are limited on understanding the regulation no one could answer the questions at the beginning. And like you said the new norm. I have done extensive research based on the FR and could have easily answered the Stop Loss question yesterday referencing the language but offering insight on a personal level rather than a regulatory level.

As for the template, I also build off the CSV Wide but using excel formulas rather than SQL (mostly due to lack of coding experience). However, I am in the process of looking into other tools that can take on more data then excel and can update easier. Currently testing out Alteryx. I want to get to a point that I can build out a file year 1 and the following years just update the CDM, Fee Schedules, DRG files and it will just update.

Can you offer any insight on the Estimated Amount payors? My major issue is trying to tie the 835 payors to a payor|plan in the MRF. I have done the approach different ways as not all 835s are the same. I have used the 835 CLP and mapped it to the insurance mnemonic. I have taken the 835 payor and mapped the name to the insurance mnemonic. My next try is to identify the code from the 837 file and map the code to a payor|plan then map that to the 835. Still working on the Loop and Segments but this this might work. Happy to share more.

But this conversation is all I want. Just open dialog on what others have tried and failed and succeeded and lessons learned. Thanks for all the help

Victoria Duffel MAFM, CRCR, CPC, CFE
Managing Consultant / Healthcare Performance Improvement
D: 816.489.4259
***@***.***
forvismazars.us<https://www.forvismazars.us/>

Follow Forvis Mazars on LinkedIn<https://www.linkedin.com/company/forvis-mazars-us/about/> | X<https://x.com/i/flow/login?redirect_after_login=%2FForvisMazarsUS> | Facebook<https://www.facebook.com/forvismazarsusa> | Instagram<https://www.instagram.com/forvis_llp/> | ***@***.***>
